TIP-0003/README.md

```diff
@@ -0,0 +1,273 @@
+
+#### Minting of Group Constructor Tokens
+
+The minting of a group constructor token requires to burn a certain amount of LVLs and to provide the policy that describes the group. This requires the submission of a minting transaction to the node. To support this kind of transactions, the following validations need to be performed on the transaction:
+
+- *Check Moving Constructor Tokens:* Let $g$ be a group identifier, then the number of Group Constructor Tokens with group identifier $g$ in the input is equal to the number of the number of Group Constructor Tokens with identifier $g$ in the output.
+- *Check Minting Constructor Tokens:* Let $g$ be a group identifier and $p$ the group policy whose digest is equal to $g$, a transaction is valid only if the all of the following statements are true:
+  - The policy $p$ is attached to the transaction.
+  - The number of group constructor tokens with identifier $g$ in the output of the transaction is strictly bigger than 0.
+  - The registration UTXO referenced in $p$ is present in the inputs and contains LVLs.
+
+#### Minting of Series Constructor Tokens
+
+The minting of a series constructor token requires to burn a certain amount of LVLs and to provide te policy that describes the series. This requires the submission of a minting transaction to the node. To support this kind of transactions, the following validations need to be performed on the transaction:
+
+- *Check Moving Series Tokens:* Let $s$ be a series identifier, then the number of Series Constructor Tokens with group identifier $s$ in the input is equal to the number of the number of Series Constructor Tokens with identifier $s$ in the output.
+- *Check Minting Constructor Tokens:* Let $s$ be a series identifier and $p$ the series policy whose digest is equal to $s$, all of the following statements are true:
+  - The policy $p$ is attached to the transaction.
+  - The number of series constructor tokens with identifier $s$ in the output of the transaction is strictly bigger than 0.
+  - The registration UTXO referenced in $p$ is present in the inputs and contains LVLs.
+
+#### Minting of an Asset Token
+
+The minting of an asset requires to use one group and one series constructor token. This requires the submission of a minting transaction to the node. To support this kind of transactions, the following validations need to be performed on the transaction.
+
+- *Check Minting of Asset Tokens:* Let $A_0$ ... $A_{n-1}$ be the $n$ Asset Minting Statements to mint $m_0$, $m_1$, ..., $m_{n-1}$  tokens with identifier  ($g$, $s$) , all of the following statements are true:
+
+  - All Asset Minting Statements $A_0$ ... $A_{n-1}$  are attached to the transaction.
+
+  - Let $in$ be the number of assets with identifier ($g$ ,$s$) in the input and $out$  number of assets with identifier ($g$ ,$s$) in the output, then: 
+    $$
+    in + \sum_{i = 1}^{n - 1} m_i = out
+    $$
+
+  - For all $A_i$, all UTXOs referenced are unique.

```

Comment:
Unique, or distinct? I feel like asking people to check for UTXO uniqueness is a slippery slope, but distinctness (from each other) is a much easier ask.

Reply:
distinct is indeed the right word